### ISSUE TITLE
Docker Pandora FMS full stack based on Centos 8 does not start

### ISSUE THREAD

**Issue by cvc90** [pandorafms/pandorafms#224]
Hello, I can't get the Docker Pandora FMS full stack based on Centos 8 version to work. In the logs, the only thing I see that gives an error are the following lines.
 
```
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 172.17.0.35. Set the 'ServerName' directive globally to suppress this message

ERROR: [pool www] failed to read the ACL of the socket '/run/php-fpm/www.sock': Operation not supported (95)

ERROR: FPM initialization failed

DEBG fd 21 closed, stopped monitoring <POutputDispatcher at 139924159658920 for <Subprocess at 139924159956920 with name php-fpm in state STARTING> (stderr)>

INFO success: php-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)

INFO exited: php-fpm (exit status 78; not expected)

DEBG received SIGCHLD indicating a child quit

INFO spawned: 'php-fpm' with pid 509

DEBG fd 15 closed, stopped monitoring <POutputDispatcher at 139924159589344 for <Subprocess at 139924159956920 with name php-fpm in state STARTING> (stdout)>

DEBG 'php-fpm' stderr output:

ERROR: [pool www] failed to read the ACL of the socket '/run/php-fpm/www.sock': Operation not supported (95)

ERROR: FPM initialization failed

```

The creation of the Docker has been done with this configuration

```
docker run --name PandoraFMS -d \
-p 8380:80 \
-p 41121:41121 \
-p 162:162/udp \
-p 9995:9995/udp \
-e DBHOST=192.168.80.45 \
-e DBNAME=pandora \
-e DBUSER=pandora \
-e DBPASS=pandora \
-e DBPORT=3306 \
-e INSTANCE_NAME=PandoraFMS01 \
-ti pandorafms-open-stack-el8
```

Anyone know what might be happening? Or what am I doing wrong?

**Comment by JimmyOlano** [comment]
¡Hi!

https://pandorafms.com/manual/start?id=en/documentation/02_installation/01_installing#minimum_software_requirements

Test PFMS on Rocky Linux 8 (CentOS 8 is not listed, but CentOS 7).

**Comment by cvc90** [comment]
The steps to follow are those described [here](https://pandorafms.com/manual/start?id=en/documentation/02_installation/01_installing#installation_under_docker) for the Docker version, what you indicate is for normal installations.

> ¡Hi!
> 
> https://pandorafms.com/manual/start?id=en/documentation/02_installation/01_installing#minimum_software_requirements
> 
> Test PFMS on Rocky Linux 8 (CentOS 8 is not listed, but CentOS 7).

**Comment by rafaelameijeiras** [comment]
Hello @cvc90 you are right for docker images should not be an issue to use CentOS 8 as a host OS.

I'll need to test it on CentOS 8, i know docker is implementing privileged containers in the latest version, so I suppose it could be something around it.

In any case, have you tested the docker compose file, which up the whole stack ? https://hub.docker.com/r/pandorafms/pandorafms-open-stack-el8

I'll do some testing in docker version over Centos 8 and get back to you.

kind regards,

**Comment by rafaelameijeiras** [comment]
@cvc90 I tested in docker over CentOS 8 stream and works as expected, could you try it again with the compose file to make sure all is running properly?  

Which docker version are you using? Seems very weird ACLs permissions for PHP just at start point.

**Comment by cvc90** [comment]
As soon as I have a moment I'll try and tell you, I think the Docker version was 20.10.3, I also tell you that after reading your comment I will check the permissions of the folder that is linked to the PandoraFMS docker.

**Comment by ejstacey** [comment]
Hi.  I hit this same issue trying to set up the stack.  I tried doing a manual setup, and also tried using the docker-compose file located here: https://github.com/pandorafms/pandorafms/blob/develop/extras/docker/centos8/docker-compose.yml

I'm running Docker version 20.10.23, build 876964a on my Synology NAS.

Every attempt I made had this issue with php-fpm erroring out.  To fix it, I replicated the changes made to /etc/php-fpm.d/www.conf in this merge request for a similar project (cacti on docker): https://github.com/scline/docker-cacti/pull/66/files

If you don't want to click through, edit /etc/php-fpm.d/www.conf, make the following changes:

Uncomment the following lines (remove the ; at the beginning):
`
listen.owner = apache
listen.group = apache
`

Comment the following line (add the ; at the beginning):
`
; listen.acl_users = apache,nginx
`

Hope this helps!
